helix-core/src/test/java/org/apache/helix/integration/task/TestTaskRebalancerStopResume.java

```diff
@@ -159,13 +190,16 @@ public void stopAndResumeNamedQueue() throws Exception {
     verifyJobNotInQueue(queueName, namespacedJob2);
   }
 
-  @Test
+  @Test(enabled = true)
```

Comment:
default is true.

  /**
   * Whether methods on this class/method are enabled.
   */
  public boolean enabled() default true;

Reply:
removed.